### ISSUE TITLE
[PATCH] generate_svn_version.sh fails on non-English systems

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#952]
**Reported by Nicolas on 25 Oct 40043588 15:33 UTC**
On my computer, generate_svn_version.sh creates a svn_version.h file containing:
```
#define SVN_VERSION "Repository: http://boinc.berkeley.edu/svn/trunk/boinc Revision: del"
```

"del"? You can probably figure out why:
```
~/src/boinc$ svn info
Ruta: .
URL: http://boinc.berkeley.edu/svn/trunk/boinc
Raz del repositorio: http://boinc.berkeley.edu/svn
UUID del repositorio: 677681c0-c3c0-48b4-bd46-92119ef6d97a
Revisin: 20177
Tipo de nodo: directorio
Agendado: normal
Autor del ltimo cambio: romw
Revisin del ltimo cambio: 20176
Fecha de ltimo cambio: 2010-01-15 15:52:58 -0300 (vie 15 de ene de 2010)
```

The script is searching for `^Rev`, and "`Revisin del ltimo cambio`" matches that.

The solution is to run it under the C locale:

```diff
Index: generate_svn_version.sh
===================================================================
--- generate_svn_version.sh     (revision 20177)
+++ generate_svn_version.sh     (working copy)
@@ -16,7 +16,7 @@
 fi

 if [ "x$CMD" != "x" ]; then
-    $CMD | awk '/^URL/ { url = $2; }; \
+    LANG=C $CMD | awk '/^URL/ { url = $2; }; \
                 /^Rev/ { rev = $2; }; \
                 END { print "#define SVN_VERSION \"Repository: " url \
                             " Revision: " rev "\""; };' \
```

**Comment by romw** [comment]
**Commented by romw on 12 Aug 40859054 20:26 UTC**
David?

**Comment by romw** [comment]
**Commented by davea on 30 Nov 40859098 09:46 UTC**
(In [22660])  - unix build system: fix for non-english locale, from Nicolas. fixes #975
